src/C-interface/dense/bml_multiply_dense_typed.c

```diff
@@ -22,6 +22,13 @@
 #include <omp.h>
 #endif
 
+#ifdef MKL_GPU
+#include "stdio.h"
+#include "mkl.h"
+#include "mkl_omp_offload.h"
+//#include "common.h"
```

Comment:
Why leave this line in there if it's commented out?

Reply:
forgot to remove it. Thanks!

Reply:
Thanks!